lib/parallel_cucumber/formatters/json_formatter.js

```diff
@@ -27,12 +28,27 @@ var JsonFormatter = function(options) {
       Debug('New feature');
       self._features.push(currentFeature);
     }
+
+    if (self.outFilePath) {
```

Comment:
@peter-reperger the file output code seems to be both in JsonFormatter and Formatter now.  Previously the file output code was only in Formatter.  Is there a way to keep the file output code in only Formatter?  Is there anything unique about the file output logic in JsonFormatter?